src/components/Highlights/Impact/index.tsx

```diff
@@ -0,0 +1,154 @@
+import { Row, createColumnHelper } from "@tanstack/react-table";
+import { useContext, useEffect, useState } from "react";
+import { PERFORMANCE_IMPACT_DOCUMENTATION_URL } from "../../../constants";
+import { usePrevious } from "../../../hooks/usePrevious";
+import { openURLInDefaultBrowser } from "../../../utils/actions/openURLInDefaultBrowser";
+import { sendUserActionTrackingEvent } from "../../../utils/actions/sendUserActionTrackingEvent";
+import { ConfigContext } from "../../common/App/ConfigContext";
+import { getImpactScoreLabel } from "../../common/ImpactScore";
+import { Button } from "../../common/v3/Button";
+import { Card } from "../../common/v3/Card";
+import { Tag } from "../../common/v3/Tag";
+import { EmptyStateCard } from "../EmptyStateCard";
+import { EnvironmentName } from "../common/EnvironmentName";
+import { Section } from "../common/Section";
+import { Table } from "../common/Table";
+import { TableText } from "../common/TableText";
+import { handleEnvironmentTableRowClick } from "../handleEnvironmentTableRowClick";
+import { trackingEvents } from "../tracking";
+import * as s from "./styles";
+import { EnvironmentImpactData } from "./types";
+import { useImpactData } from "./useImpactData";
+
+const getRankTagType = (rankCriticality: number) => {
+  if (rankCriticality >= 0.9) {
```

Comment:
For insights, we use 0.8, should it be the same or it calculated diffierently?

Reply:
Yes, the calculation is a bit different: https://github.com/digma-ai/digma-collector-backend/issues/1862